script.js

```diff
@@ -0,0 +1,298 @@
+      en: {
+        nocaps: [
+          '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+          'Tab', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\', 'Del',
+          'Caps Lock', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', "'", 'Enter',
+          'Shift', 'z', 'x', 'c', 'v', 'b', 'n', 'm', '.', ',', '/', '\u2191', 'Shift',
+          'Ctrl', 'Win', 'Alt', ' ', 'Alt', '\u2190', '\u2193', '\u2192', 'Ctrl',
+        ],
+        caps: [
+          '~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+', 'Backspace',
+          'Tab', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|', 'Del',
+          'Caps Lock', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ';', '"', 'Enter',
+          'Shift', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?', '\u2191', 'Shift',
+          'Ctrl', 'Win', 'Alt', ' ', 'Alt', '\u2190', '\u2193', '\u2192', 'Ctrl',
+        ],
+      },
+      ru: {
+        nocaps: [
+          'ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', 'Backspace',
+          'Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'Del',
+          'Caps Lock', 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', 'Enter',
+          'Shift', 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '/', '\u2191', 'Shift',
+          'Ctrl', 'Win', 'Alt', ' ', 'Alt', '\u2190', '\u2193', '\u2192', 'Ctrl',
+        ],
+        caps: [
+          'Ё', '!', '"', '№', '%', ':', ',', '.', ';', '(', ')', '_', '+', 'Backspace',
+          'Tab', 'Й', 'Ц', 'У', 'К', 'Е', 'Н', 'Г', 'Ш', 'Щ', 'З', 'Х', 'Ъ', '|', 'Del',
+          'Caps Lock', 'Ф', 'Ы', 'В', 'А', 'П', 'Р', 'О', 'Л', 'Д', 'Ж', 'Э', 'Enter',
+          'Shift', 'Я', 'Ч', 'С', 'М', 'И', 'Т', 'Ь', 'Б', 'Ю', '?', '\u2191', 'Shift',
+          'Ctrl', 'Win', 'Alt', ' ', 'Alt', '\u2190', '\u2193', '\u2192', 'Ctrl',
+        ],
+      },
+    };
+    this.hardKeys = [
+      'Backquote', 'Digit1', 'Digit2', 'Digit3', 'Digit4', 'Digit5', 'Digit6', 'Digit7', 'Digit8', 'Digit9', 'Digit0', 'Minus', 'Equal', 'Backspace',
+      'Tab', 'KeyQ', 'KeyW', 'KeyE', 'KeyR', 'KeyT', 'KeyY', 'KeyU', 'KeyI', 'KeyO', 'KeyP', 'BracketLeft', 'BracketRight', 'Backslash', 'Delete',
+      'CapsLock', 'KeyA', 'KeyS', 'KeyD', 'KeyF', 'KeyG', 'KeyH', 'KeyJ', 'KeyK', 'KeyL', 'Semicolon', 'Quote', 'Enter',
+      'ShiftLeft', 'KeyZ', 'KeyX', 'KeyC', 'KeyV', 'KeyB', 'KeyN', 'KeyM', 'Comma', 'Period', 'Slash', 'ArrowUp', 'ShiftRight',
+      'ControlLeft', 'MetaLeft', 'AltLeft', 'Space', 'AltRight', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'ControlRight',
+    ];
+  }
+
+  init(wrp, out) {
+    const outputElement = out;
+    this.elements.keysContainer = document.createElement('div');
+    this.elements.keysContainer.classList.add('keyboard');
+    wrp.appendChild(this.elements.keysContainer);
+    const curLang = window.sessionStorage.getItem('lang');
+    this.properties.lang = (curLang == null) ? 'en' : curLang;
```

Comment:
Can be refactored to:
`this.properties.lang = window.sessionStorage.getItem('lang') || 'en';`